x-pack/docs/en/security/fips-140-compliance.asciidoc

```diff
@@ -6,8 +6,11 @@ The Federal Information Processing Standard (FIPS) Publication 140-2, (FIPS PUB
 140-2), titled "Security Requirements for Cryptographic Modules" is a U.S.
 government computer security standard used to approve cryptographic modules.
 {es} offers a FIPS 140-2 compliant mode and as such can run in a FIPS 140-2
-enabled JVM. In order to set {es} in fips mode, you must set the
-`xpack.security.fips_mode.enabled` to `true` in `elasticsearch.yml`
+configured JVM.
+
+After ensuring that your JVM is properly configured according to FIPS 140-2 requirements
+using the Java Security Provider of your choice, you can set {es} in fips mode
+by setting the `xpack.security.fips_mode.enabled` to `true` in `elasticsearch.yml`
```

Comment:
Do we want to explicitly say that the JVM that is bundled with ES, is not configured to run with a FIPS certified security provider?

Reply:
Good point, I'll add the text.